docs/source/dev_guide/release.rst

```diff
@@ -72,7 +72,7 @@ Releasing on GitHub: release candidates
 
 	$ git checkout main
 	$ git fetch upstream
-	$ git rebase upstream/main
+	$ git reset --hard upstream/main
 	$ git tag -a v1.1.0rc1 -m "v1.1.0rc1"
 	$ git push upstream v1.1.0rc1
```

Comment:
Delete the branch from the `tbump` step first. Otherwise this line will fail because the branch has the same name.